airflow/www/fab_security/manager.py

```diff
@@ -390,7 +390,7 @@ def my_oauth_user_info(sm, provider, response=None):
         def wraps(provider, response=None):
             ret = f(self, provider, response=response)
             # Checks if decorator is well behaved and returns a dict as supposed.
-            if not type(ret) == dict:
+            if not isinstance(ret, dict):
```

Comment:
It's slightly different meaning, but yes - that's how it should be, so  this is really a bugfix (for example the first check would not work if we returned OrderedDict, but the fix would)